cosinnus_message/rocket_chat.py

```diff
@@ -467,13 +464,18 @@ def users_create_or_update(self, user, request=None):
         else:
             return self.users_create(user, request)
 
+    def has_username_changed(self, profile, rocket_user_name):
+        """Check it username needs to be updated in RocketChat."""
+        username = profile.get_new_rocket_username()
+        username_match = re.match(rf'^{username}\d*$', rocket_user_name)
```

Comment:
1.1 This regex now needs to be adjusted to the changes in `profile.get_new_rocket_username()`, where "-<userid>" is now appended to newly generated usernames.

1.2. Also as a question: after we deploy this, this would cause all old-style rocketchat usernames (without userid) to be changed to the new style (with userid) whenever an old userprofile is saved, right? Do you think this could have any implications or side-effects?

Reply:
> 1.1

Right. I'll adjust it.

> 1.2

Yes. The usernames will be updated after a login. I can't think of any implications.